src/test/java/org/apache/maven/buildcache/checksum/MavenProjectInputTest.java

```diff
@@ -47,52 +47,23 @@
 
 public class MavenProjectInputTest {
 
-    private static final String GLOB = "{*-pom.xml}";
+    private static final String DEFAULT_GLOB = "*";
 
     @Test
-    public void testAddInputsRelativePath() {
-        //        MavenProjectInput inputs = new MavenProjectInput(config, new ArrayList<Path>(),
-        // Paths.get("src\\test\\resources\\org"), GLOB);
-        //        ArrayList<Path> files = new ArrayList<>();
-        //        inputs.listDirOrFile("../../resources", inputs.dirGlob, files, new HashSet<Path>());
-        //        assertEquals(4, files.size());
-    }
-
-    @Test
-    public void testAddInputsAbsolutePath() {
-        //        Path baseDirPath = Paths.get("src\\test\\resources\\org");
-        //        MavenProjectInput inputs = new MavenProjectInput(config, new ArrayList<Path>(), baseDirPath, GLOB);
-        //        ArrayList<Path> files = new ArrayList<>();
-        //        Path candidatePath = baseDirPath.resolve("../../resources").normalize().toAbsolutePath();
-        //        inputs.listDirOrFile(candidatePath.toString(), inputs.dirGlob, files, new HashSet<Path>());
-        //        assertEquals(4, files.size()); // pom is filtered out by hardcoded if
-    }
-
-    @Test
-    public void testGlobInput() {
-        //        Path baseDirPath = Paths.get("src\\test\\resources");
-        //        MavenProjectInput inputs = new MavenProjectInput(config, new ArrayList<Path>(), baseDirPath, GLOB);
-        //        ArrayList<Path> files = new ArrayList<>();
-        //        inputs.tryAddInputs("*.java", files, new HashSet<Path>());
-        //        assertEquals(0, files.size()); // pom is filtered out by hardcoded if
-    }
-
-    @Test
-    public void testGetDirectoryFiles() {
-        List<Path> directoryFiles = new ArrayList<>();
+    public void testCollectFilteredFiles() {
```

Comment:
Cleaned dead code and updated a test with a wrong assumption. Not the most useful test but better than before.